packages/vitest/src/node/cli.ts

```diff
@@ -101,9 +101,6 @@ async function run(cliFilters: string[], options: UserConfig) {
     if (!ctx.config.watch)
       await ctx.close()
   }
-
-  if (!ctx.config.watch)
-    process.exit()
```

Comment:
If we don't have this, the process might be hanging due to unresolved side effects

Reply:
Terminating the process causes the crash too

Reply:
Did you have an issue with unresolved effects or this is just a precaution?

Reply:
NodeJs workers are really fragile. If there anything going on inside worker (open port, file handle), terminating it causes a crash.

Reply:
@Demivan  With your changes windows will run ok, testing this last change (3 consecutive `nr ci` + 3 sequences alternating `pnpm run test:cli` and `pnpm run test:all`)...

Reply:
It could be dangerous as a hanging process will eat up all your CI time 👀. At some point, there will be some cases, but not very sure if it still applies since we migrate to workers (but there is still a non-worker mode). Could never be too prepared :P